### PR TITLE
PIL-1140 Switch to in-app browser for WC Dashboard

### PR DESCRIPTION
**Pull request by divyaBacancy**

### CODE REVIEW

File: src/screens/WalletConnect/Home/WalletConnectHome.js [hunk 1]
```diff
@@ -67,6 +73,7 @@ function WalletConnectHome() {
   const navigation = useNavigation();
   const safeArea = useSafeAreaInsets();
   const isReady = useInteractionManager(); // Used to prevent jank on screen entry animation
+  const enableURLInAppBrowser = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_WC_DASHBOARD_INAPPBROWSER);
```
Comment (IAmKio) [inline src/screens/WalletConnect/Home/WalletConnectHome.js:76]: Is this supposed to be `.getBoolean` or something of that sort?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a small thing for you to check Divya but aside from that, it looks good to me!

**Comment by IAmKio** [comment]
@divyaBacancy so sorry, can we change the default value of the feature switch to false for now? Thank you

**Comment by divyaBacancy** [comment]
@IAmKio Done.

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**